src/components/ui4/ModalUi4.tsx

```diff
@@ -120,14 +117,16 @@ export function ModalUi4<T>(props: ModalPropsUi4<T>): JSX.Element {
     borderColor: warning ? theme.warningText : theme.modalBorderColor,
     borderWidth: warning ? 4 : theme.modalBorderWidth,
     marginBottom: -bottomGap,
-    paddingTop: isHeaderless ? customPadding.paddingTop : 0, // If there's a header; either close button or a title, the custom paddingTop will be added to the bottom of the title container.
-    paddingBottom: bottomGap + (scroll ? 0 : customPadding.paddingBottom), // Ignore custom padding on bottom for scrollable content so we don't have a cutoff gap when scrolling content
+    paddingTop: isHeaderless ? halfRem : 0, // If there's a header; either close button or a title, the custom paddingTop will be added to the bottom of the title container.
+    paddingBottom: bottomGap + (scroll ? 0 : halfRem), // Ignore padding on bottom for scrollable content so we don't have a cutoff gap when scrolling content
 
     // No matter if we are scrollling or not, horizontal paddings are fixed
-    paddingLeft: customPadding.paddingLeft,
-    paddingRight: customPadding.paddingRight
+    paddingLeft: halfRem,
+    paddingRight: halfRem
   }
 
+  const titlePadding = { marginBottom: halfRem }
+
```

Comment:
It looks like this commit simply removed the custom padding. That's great! However, now that this is a constant, it could be baked into `styles.titleContainer`